x-pack/plugins/stack_alerts/server/alert_types/geo_threshold/tests/es_sample_response_with_nesting.json

```diff
@@ -0,0 +1,211 @@
+{
+  "took" : 2760,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 10000,
+      "relation" : "gte"
+    },
+    "max_score" : 0.0,
+    "hits" : [
```

Comment:
do we need to include `hits` for testing this?

also, if we do, shouldn't we need to update `@timestamp` to use `time_data.@timestamp` instead for clarity?

Reply:
Very likely not. This is a copy of an old response before we tweaked the query to not include `hits`. I'll remove it from both files :+1: